msal/build.gradle

```diff
@@ -125,9 +125,9 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.5', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '0.0.6', changing: true)
 
-    distApi("com.microsoft.identity:common:0.0.5") {
+    distApi("com.microsoft.identity:common:0.0.6") {
```

Comment:
same here : `0.0.7`

Reply:
Thanks, resolved in d0bb6d1045c8c80f3aacb4b054ed45cf0c9cce0c